### ISSUE TITLE
Lambda value

### ISSUE THREAD

**Issue by khawar-islam** [hzlsaber/IPMix#4]
Dear @hzlsaber 

One again thanks for your contribution. As I am working on your code and I have a question for you that how do you choose lambda value 12?
```
loss += 12 * (F.kl_div(p_mixture, p_clean, reduction='batchmean') +
                          F.kl_div(p_mixture, p_aug1, reduction='batchmean') +
                          F.kl_div(p_mixture, p_aug2, reduction='batchmean')) / 3.
```

**Comment by hzlsaber** [comment]
Thank you for your question. 

This value was initially chosen based on insights from prior research[1] in similar domains, where balancing different components of the loss function is crucial. It's an empirical choice that worked well in our experiments.

However, this value is not fixed and might not be optimal for all scenarios. For example,[2] sets the lambda value of 10 and works well. I encourage you to experiment with different lambda values to see how it impacts your specific use case. Adjusting this value could lead to interesting findings and potentially improve the model's performance on your dataset.

[1]Hendrycks, Dan et al. “AugMix: A Simple Data Processing Method to Improve Robustness and Uncertainty.” ArXiv abs/1912.02781 (2019): n. pag.
[2] Wang, Haotao et al. “AugMax: Adversarial Composition of Random Augmentations for Robust Training.” Neural Information Processing Systems (2021).

**Comment by khawar-islam** [comment]
Always thanks for your swift answer and references
